fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -141,4 +141,5 @@
     <include file="parts/0119_add_loan_product_down_payment_configuration.xml" relativeToChangelogFile="true" />
     <include file="parts/0120_transaction_summary_with_asset_owner_report_typo_fix.xml" relativeToChangelogFile="true" />
     <include file="parts/0121_transaction_summary_with_asset_owner_report_typo_fix_2.xml" relativeToChangelogFile="true" />
+    <include file="parts/0122_add_loan_product_auto_repayment_down_payment_configuration.xml" relativeToChangelogFile="true" />
```

Comment:
Please move this into the fineract-loan module. You can find a couple example for liquibase changes.